CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.5.16
```

Comment:
```suggestion
## 0.6.0
```

Per https://semver.org/ if new functionality was added minor version should be incremented.
Patch version is for fixes.

Reply:
Can you include this change as well?